fvm/systemcontracts/system_contracts.go

```diff
@@ -0,0 +1,178 @@
+// Package systemcontracts stores canonical address locations for all system
+// smart contracts and service events.
+//
+// System contracts are special smart contracts controlled by the service account,
+// a Flow account with special privileges to administer the network.
+//
+// Service events are special events defined within system contracts which
+// are included within execution receipts and processed by the consensus committee
+// to enable message-passing to the protocol state.
+//
+// For transient networks, all system contracts can be deployed to the service
+// account. For long-lived networks, system contracts are spread across several
+// accounts for historical reasons.
+package systemcontracts
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+const (
+
+	// Unqualified names of system smart contracts (not including address prefix)
+
+	ContractNameEpoch     = "FlowEpoch"
+	ContractNameClusterQC = "FlowClusterQC"
```

Comment:
The actual name of the contract is `FlowEpochCusterQC`, but that makes me realize that it would probably be good to have consistent naming of the contracts. Should it be `FlowEpochClusterQC` and `FlowEpochDKG` or just `FlowClusterQC` and `FlowDKG`?

Reply:
I'd go with `FlowClusterQC` and `FlowDKG`, but is it a simple change to rename the contract?

Reply:
It should be simple, just gotta go through the contracts and transactions and find and replace